index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/105bcdf182.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <title>Hello World</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <a class="logo">Habib Payenda</a>
+            <i id="menu" class="fa-solid fa-bars"></i>
+        </nav>
+    </header>
+    <section class="headline">
+        <div class="text-holder">
+            <h1 class="primary-text">I'm Habib glad to see you!</h1>
+            <p class="details">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Omnis deserunt facilis tempore cupiditate. Reprehenderit, voluptatibus nam corrupti omnis quam vero, id minus ea quae in aperiam eum, veritatis aliquid non.
+            </p>
+            <a class="connect">Let's connect</a>
+        </div>
+        <div class="fa-container">
+            <i class="fa-brands fa-twitter fa"></i>
+            <i class="fa-brands fa-linkedin-in fa"></i>
+            <i class="fa-brands fa-medium fa" ></i>
+            <i class="fa-brands fa-github fa"></i>
+            <i class="fa-solid fa-hand-peace fa"></i>
+        </div>
+    </section>
+</body>
+</html>
```

Comment:
Your application is not matching the Figma template of your choosing. You need to use the same background image, same margins and padding and follow the exact layout as in the Figma template.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/176422389-4c9c56dd-4aca-4f23-a478-531406e3ae75.png) | ![image](https://user-images.githubusercontent.com/33268927/176422255-4d37a205-6c96-4a89-a559-100cce563687.png) |

Reply:
Where can I get that background image from?

Reply:
Form the Figma template.